src/plugins/GUITestBase/src/GUITestBasePlugin.cpp

```diff
@@ -1798,6 +1798,7 @@ void GUITestBasePlugin::registerTests(UGUITestBase* guiTestBase) {
     REGISTER_TEST(GUITest_regression_scenarios::test_7609);
     REGISTER_TEST(GUITest_regression_scenarios::test_7616);
     REGISTER_TEST(GUITest_regression_scenarios::test_7623);
+    REGISTER_TEST_WITH_TIMEOUT(GUITest_regression_scenarios::test_7629, 720000);
```

Comment:
You know that I will never approve this right? Make the test smart and fast. (<1 min)